nock/communication/send-email/mailchimp/SendEmail/recording-69d92e19523ced971d75fcf53bc8882e.json

```diff
@@ -35,7 +35,7 @@
     ],
     "rawHeaders": [
       "server",
-      "nginx/1.12.2",
+      "nginx/1.4.6 (Ubuntu)",
```

Comment:
Is this intended change? Looks like it was part of different PR.

Reply:
You are right, this shouldn't be there. Good catch!